### PR TITLE
feat: String based api support (Clients, Bundles, Distributions, Labels, Machine Translations)

### PR DESCRIPTION
**Pull request by Zahid92**
Yet to complete, Initially I was trying to get it done in a single chunk, realized my mistake and now will implement in small chunks one thing at a time
#214

### CODE REVIEW

**Comment by andrii-bodnar** [comment]
@Zahid92 thank you! Please let me know when I can start reviewing this PR.

**Comment by andrii-bodnar** [comment]
@Zahid92 Thanks for letting me know! 

Could you please summarize in the original post what has been done and what still needs to be done?

**Comment by Zahid92** [comment]
Hi @andrii-bodnar, here the list for whats done and whats remaining.

String based support for : 
- [x]  Clients API
- [x] Bundles
- [x] Distributions
- [x] Labels
- [x] Machine Translations

Remaining String based support for:
- [ ] Notifications api
- [ ] Projects api
- [ ] Reports api
- [ ] String translations api
- [ ] Tasks api
- [ ] Teams api
- [ ] Translation memory api
- [ ] Translation status api
- [ ] Translations api
- [ ] Users api
- [ ] Vendors api
- [ ] Webhooks api
- [ ] Strings api

**Comment by andrii-bodnar** [comment]
@Zahid92 thank you!

File: src/Crowdin.Api/Bundles/BundlesApiExecutor.cs [hunk 1]
```diff
@@ -1,47 +1,62 @@
...
             _jsonParser = jsonParser;
         }
-        
+
         /// <summary>
         /// List bundles. Documentation:
-        /// <a href="https://developer.crowdin.com/api/v2/#operation/api.projects.bundles.getMany">Crowdin API</a>
-        /// <a href="https://developer.crowdin.com/enterprise/api/v2/#operation/api.projects.bundles.getMany">Crowdin Enterprise API</a>
+        /// <a href="https://developer.crowdin.com/api/v2/string-based/#operation/api.projects.bundles.branches.getMany">Crowdin API</a>
+        /// <a href="https://developer.crowdin.com/enterprise/api/v2/string-based/#operation/api.projects.bundles.branches.getMany">Crowdin Enterprise API</a>
```
Comment (andrii-bodnar) [inline src/Crowdin.Api/Bundles/BundlesApiExecutor.cs:31]: Wrong links. It shouldn't point to the Bundle List Branches API. It's probably inadvertently linked with the method below

File: src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs [hunk 2]
```diff
@@ -52,6 +52,19 @@ public async Task<Distribution> AddDistribution(int projectId, AddDistributionRe
             return _jsonParser.ParseResponseObject<Distribution>(result.JsonObject);
         }
 
+        /// <summary>
+        /// Add distribution. Documentation:
+        /// <a href="https://support.crowdin.com/api/v2/#operation/api.projects.distributions.post">Crowdin API</a>
+        /// <a href="https://support.crowdin.com/enterprise/api/#operation/api.projects.distributions.post">Crowdin Enterprise API</a>
```
Comment (andrii-bodnar) [inline src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs:58]: It's a file-based documentation links

File: src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs [hunk 3]
```diff
@@ -116,6 +142,17 @@ public async Task<DistributionRelease> ReleaseDistribution(int projectId, string
             CrowdinApiResult result = await _apiClient.SendPostRequest(url);
             return _jsonParser.ParseResponseObject<DistributionRelease>(result.JsonObject);
         }
+        /// <summary>
+        /// Release distribution. Documentation:
+        /// <a href="https://support.crowdin.com/api/v2/#operation/api.projects.distributions.release.post">Crowdin API</a>
```
Comment (andrii-bodnar) [inline src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs:147]: should be string-based docs

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

File: src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs (hunk 2 above)
Comment (Zahid92) [reply src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs:58]: For enterprise is there string-based documentation available ?

File: src/Crowdin.Api/Bundles/BundlesApiExecutor.cs (hunk 1 above)
Comment (Zahid92) [reply src/Crowdin.Api/Bundles/BundlesApiExecutor.cs]: Updated

File: src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs (hunk 3 above)
Comment (Zahid92) [reply src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs]: Updated

File: src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs (hunk 2 above)
Comment (andrii-bodnar) [reply src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs]: Not yet

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**